docs/tutorials/task.csv-to-oscal-cd/csv-to-oscal-cd.md

```diff
@@ -0,0 +1,151 @@
+# task csv-to-oscal-cd
+
+The below table represents the expectations of trestle task `csv-to-oscal-cd` for the contents of the input csv-file for synthesis of the output OSCAL Component Definition json-file.
+
+`Column Name` is the name of the expected column in the input csv-file. Any additional columns not identified here, for example foobar, are also extracted and placed into the output json-file as component.control-implementation.prop\["foobar"\].
+
+`Component Definition Locale` is the `path` within the output json-file into witch the value is stashed.
+
+An example input [csv-file](ocp4-sample-input.csv).
+
+An example output [json-file](component-definition.json).
+
+<table>
+
+<tr style="text-align:left;vertical-align:top">
+<th>Column Name
+<th>Value Type
+<th>Specification
+<th>Value Description
+<th>Component Definition Locale
+<th>Example Value
+
+<tr style="text-align:left;vertical-align:top">
+<td>Rule_Id
+<td>String
+<td>required
+<td>A textual label that uniquely identifies a policy (desired state) that can be used to reference it elsewhere in this or other documents.
```

Comment:
Its not a "label" but a "name" for a rule.
"policy (desired state).." to "policy rule (desired...."

Reply:
@vikas-agarwal76 Can we leave these text as-is for now and make any fixups with #1233 ?

Reply:
ok